app/services/user_merger.rb

```diff
@@ -30,6 +31,9 @@ def merge!
   protected
 
   def update_username
+    return if @source_user.username == @target_user.username
+
+    ::MessageBus.publish '/merge_user', { message: I18n.t("admin.user.merge_user.updating_username") }
```

Comment:
I think the `MessageBus.publish` calls should be scoped to the `acting_user` or is there a reason for those messages to be available for all users?

Reply:
Oh very good catch @gschlager ! Yes this must be fixed.